libs/shared-atomic-design-components/src/lib/atoms/CardTags.scss

```diff
@@ -1,21 +1,63 @@
-.tags {
-  margin-top: 1rem;
+@import '_variables.scss';
 
-  .tag {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+.wrapper {
+  &__desktop {
+    display: flex;
+    align-items: flex-start;
+    gap: 16px;
+
+    .chip {
+      display: flex;
+      height: 32px;
+      padding: 3px 12px;
+      justify-content: center;
+      align-items: center;
+      gap: 8px;
+      flex-wrap: nowrap;
 
-    display: inline-block;
-    line-height: 2rem;
+      text-wrap: nowrap;
+      border-radius: 8px;
+      background: var(--redi-orange-extra-light, #ffeae2);
 
-    &--rest {
-      flex-shrink: 0;
+      color: var(--redi-orange, #ff7d55);
+      font-family: Avenir LT Std;
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 150%; /* 21px */
     }
   }
 
-  &__last-row {
+  &__mobile {
     display: flex;
-    width: 100%;
+    gap: 8px;
+    flex-wrap: wrap;
+
+    .last_row {
+      width: 100%;
+    }
+
+    .chip {
+      display: inline-block;
+      height: 24px;
+      padding: 3px 12px;
+      justify-content: center;
+      align-items: center;
+      gap: 8px;
+      flex-wrap: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+
+      border-radius: 6px;
+      background: var(--redi-orange-extra-light, #ffeae2);
+
+      color: var(--redi-orange, #ff7d55);
+      font-family: Avenir LT Std;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 150%; /* 21px */
+    }
```

Comment:
These styles seem repeated to me. Is there any way we can avoid this repetition?

Reply:
good catch! just removed the reduntant styles ✅